fritzconnection/lib/fritzhosts.py

```diff
@@ -148,6 +148,19 @@ def set_host_name(self, mac_address, name):
         }
         self._action('X_AVM-DE_SetHostNameByMACAddress', arguments=args)
 
+    def get_host_name(self, mac_address):
+        """
+        Returns a String with the host_name of the device with the given mac_address
+        """
+        host = list
+        try:
+            host = self.get_specific_host_entry(mac_address)
+        except:
```

Comment:
Please never use a 'naked' except, because it will be hard to find bugs later on. Explicitly catch just the error types you expect and are able to handle.